flexxamples/howtos/dynamic_container_app.py

```diff
@@ -0,0 +1,80 @@
+from flexx import flx
```

Comment:
Could you please add a short docstring at the top of the examples to briefly explain the purpose of the examples?

Reply:
Yes, will do.